tests/runtime/engine/runner.py

```diff
@@ -284,6 +285,14 @@ def run_test(test):
             if after and after.poll() is None:
                 os.killpg(os.getpgid(after.pid), signal.SIGKILL)
 
+            if test.cleanup:
+                try:
+                    cleanup = subprocess.run(test.cleanup.split(), shell=True, capture_output=True, text=True)
+                    cleanup.check_returncode()
+                except subprocess.CalledProcessError as e:
+                    print('\tCLEANUP error: %s' % e.stderr)
```

Comment:
Pls make output format consistent with other failure messages:
```suggestion
                    print(fail("[ FAILED ] ") + "%s.%s" % (test.suite, test.name))
                    print('\tCLEANUP error: %s' % e.stderr)
```

Reply:
Ack